interfaces/console/src/app/configure/node/[id]/site/[name]/install/page.tsx

```diff
@@ -39,7 +39,8 @@ const SiteConfigure = ({ params }: ISiteConfigure) => {
   const searchParams = useSearchParams();
   const qpPower = searchParams.get('power') ?? '';
   const qpSwitch = searchParams.get('switch') ?? '';
-  const stepTracker = searchParams.get('step') ?? '1';
+  const flow = searchParams.get('flow') ?? 'onb';
+  const step = parseInt(searchParams.get('step') ?? '1');
   const qpbackhaul = searchParams.get('backhaul') ?? '';
 
   const formik = useFormik({
```

Comment:
Why is this formik empty ?

Reply:
Its not empty, I think you didn't expanded the bottom code.